ui/src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
Why is this here? 😳

Reply:
This needs to be there to support importing SVGs. It brings in the SVG types etc.

Reply:
Ok, I will take your word for it :-)

Reply:
Well you can try erasing it :)

Reply:
Actually here https://stackoverflow.com/a/71153426